changelog.d/20240216_122016_derek_gcs_endpoint_update_sc_28586.md

```diff
@@ -0,0 +1,6 @@
+
+### Enhancements
+
+* [sc-28586](https://app.shortcut.com/globus/story/28586/cli-implement-globus-gcs-endpoint-update) Added support for GCSv5 endpoint displaying & updating:
```

Comment:
```suggestion
* Added support for GCSv5 endpoint displaying & updating:
```

Because the CLI changelog is end-user facing, we should not include links to our internal tracking here. It invites users to click on a link they can't follow.

Reply:
Sure.